packages/overlay/src/integrations/sentry/components/developerInfo/EnvelopeDetails.tsx

```diff
@@ -17,7 +17,7 @@ export default function EnvelopeDetails({ data }: { data: { envelope: Envelope;
           <>
             {header.event_id && (
               <>
-                Event Id: <span className="text-primary-500">&mdash;</span> {header.event_id}
+                Event Id <span className="text-primary-500">&mdash;</span> {header.event_id}
```

Comment:
intentional change?

Reply:
Yes, we have  "-" after label "Event Id". So its redundant.